omics_orientation/omics_orientation.md

```diff
@@ -0,0 +1,195 @@
+
+Genomics vs. genetics
+=====
+
+Though they are often mixed up or used interchangeably, genomics and genetics are distinct fields with different goals. Whereas genomics is the study of an organism's genome, or all of its genetic information, **genetics** is the study of specific genes (or parts of genes), their effects, and how they are inherited.
+
+What about bioinformatics? Is that the same thing?
+=====
+
+Bioinformatics is another word that is often used interchangeably with genomics, but they are also not the same thing! Bioinformatics specifically refers to "using computer technology to collect, store, analyze and disseminate biological data and information, such as DNA and amino acid sequences or annotations about those sequences" (https://www.genome.gov/genetics-glossary/Bioinformatics). Bioinformatics makes the study of genomics and other omics disciplines possible.
+
+### Transciptomics
+
+An organism's **transcriptome** is all of the RNA that is transcribed from that organism's DNA, and **transcriptomics** is the qualitative (which transcripts are present) and quantitative (how much of each transcript is present) study of these RNA transcripts. While some RNA transcripts are templates to be translated into proteins, it is becoming increasingly recognized that many RNA molecules have important biological roles beyond encoding proteins! And while every cell in an organism has the same genome, the transcriptome will vary from between cell populations and can change in response to changes in the environment, such as disease or treatment. The primary technologies for studying transcriptomics are **microarray**, in which RNA fragments of known sequences are bound to a surface (or "chip") and the amount of each fragment is quantified, and **RNA-Seq**, in which RNA sequences (which can be known or unknown) are analyzed qualitatively and quantitatively using **next-generation sequencing** (in which millions of sequences can be run at one time).
```

Comment:
```suggestion
An organism's **transcriptome** is all of the RNA that is transcribed from that organism's DNA, and **transcriptomics** is the qualitative (which transcripts are present) and quantitative (how much of each transcript is present) study of these RNA transcripts. While some RNA transcripts are templates to be translated into proteins, it is becoming increasingly recognized that many RNA molecules have important biological roles beyond encoding proteins! 


While every cell in an organism has the same genome, the transcriptome will vary from between cell populations and can change in response to changes in the environment, such as disease or treatment. The primary technologies for studying transcriptomics are **microarrays**, in which RNA fragments of known sequences are bound to a surface (or "chip") and the amount of each fragment is quantified, and **RNA-Seq**, in which RNA sequences (which can be known or unknown) are analyzed qualitatively and quantitatively using **next-generation sequencing** (in which millions of sequences can be run at one time).
```

Reply:
I like this separated, I'll add a bit to the end of the first pieces to make the transition a little smoother.